### ISSUE TITLE
AP reduced to 0.606 when NMS=False on VOC17 test set

### ISSUE THREAD

**Issue by reactivetype** [xuannianz/keras-CenterNet#22]
I have measured the AP@IoU 0.5 for the pretrained model you shared for PascalVOC. 

Measured on VOC17TEST, the results are `mAP: 0.7293` when `nms=True` and it reduces to  `mAP: 0.60` when `nms=False`.

Could you observe similar results? In the paper, they did experiments on CoCo and the reduction in mAP was quite small when NMS was not used. 

I wonder if this performance is due to implementation or a limitation of the model itself.

**Comment by reactivetype** [comment]
@xuannianz by the way, did you train the model with multi_scale training?
